### PR TITLE
middle_section

### LINKED ISSUE #2: Middle Section - Portfolio
**Make a similar middle section as shown in the screenshot**

Screenshot 2023-05-13 at 17 55 19

Screenshot 2023-05-13 at 17 57 31

### PR DESCRIPTION
**Pull request by Ramdev-Lodhi**
Screenshot (28)
Screenshot (29)

### CODE REVIEW

**Comment by Ramdev-Lodhi** [comment]
@MohitGupta121 PTAL. thanks

**Review by MohitGupta121 — APPROVED** [review]
@rsthakur8815 LGTM, Thanks

File: middle_section.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```
Comment (MohitGupta121) [inline middle_section.html]: Please change this file to index.html so that at last we combine all files.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Comment by Ramdev-Lodhi** [comment]
@MohitGupta121 PTAL. Thanks

**Pull request closed without merging**